aspnetcore/blazor/multi-project content support.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create a razor class library with static assets
+author: javiercn
+description: Learn how to create a razor class library (RCL) with static assets.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 06/26/2019
+uid: blazor/multi-project-content-support.md
+---
+
+# Create a razor class library with static assets
+
+Razor Class Libaries (RCL) frequently require companion static assets that can be to be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available from a consuming app.
```

Comment:
> that can be to be referenced

This have to be corrected

Reply:
@Andrzej-W I'll take care of the editing.